lib/common.js

```diff
@@ -121,6 +121,7 @@ class DocumentLocaleSettings {
 			}
 		}
 		this._overrides = val;
+		this._listeners.forEach((cb) => cb());
```

Comment:
So I wasn't sure of a completely ideal way to do all this. Since `overrides`, `language`, and `fallbackLanguage` can technically get changed through `.fallbackLanguage` etc., they should call the listeners, so I didn't want to duplicate that in `_handleObserverChange`, but it feels a bit weird to have two different ways that the calls are happening.